### DISCUSSION TITLE
Glue Schema Registry Deserialization - Fallback serde was used

### DISCUSSION THREAD

**Discussion by fklezin** [kafbat/kafka-ui#438] [Q&A]
I'm trying to set up Glue schema registry with AVRO deserialization and I encountered the following issue in the UI: "Fallback serde was used."
![image](https://github.com/kafbat/kafka-ui/assets/9782834/2cfce23a-37fc-47b5-a94e-4e505fa295dd)

Looking at the logs, I don't see any errors.

However, based on the code, I should at least see:

`Error deserializing key for value topic: {}`

Here is the relevant code snippet from the repo:
``` 
private void fillValue(TopicMessageDTO message, ConsumerRecord<Bytes, Bytes> rec) {
    if (rec.value() == null) {
      return;
    }
    try {
      var deserResult = valueDeserializer.deserialize(
          new RecordHeadersImpl(rec.headers()), rec.value().get());
      message.setContent(deserResult.getResult());
      message.setValueSerde(valueSerdeName);
      message.setValueDeserializeProperties(deserResult.getAdditionalProperties());
    } catch (Exception e) {
      log.trace("Error deserializing key for value topic: {}, partition {}, offset {}, with serde {}",
          rec.topic(), rec.partition(), rec.offset(), valueSerdeName, e);
      var deserResult = fallbackValueDeserializer.deserialize(
          new RecordHeadersImpl(rec.headers()), rec.value().get());
      message.setContent(deserResult.getResult());
      message.setValueSerde(fallbackSerdeName);
    }
  }
```

**I checked CloudTrail and found a successful API call to the correct schema registry and schema name. However, I only see a GetSchema call that contains schema metadata but not the actual schema definition. I was also expecting to see a GetSchemaVersion API call.**

Any suggestions on how I should troubleshoot this issue?

**Comment by MiLabuda** [comment]
Hello @fklezin 
Did you found the answer to your problem, because I'm struggling with something similar and was wondering if you've succesfully resolvved your issue.
